### PR TITLE
Fix local error messages

### PR DESCRIPTION
**Pull request by iMicknl**
Follow up on https://github.com/iMicknl/python-overkiz-api/pull/843#issuecomment-1556157440.

@shypike can you verify if this is the right fix?

### CODE REVIEW

**Comment by shypike** [comment]
It should be OK, but I'll add it to my own installation. And I will try to catch the error condition.
I can give you feedback tomorrow.

**Comment by iMicknl** [comment]
> It should be OK, but I'll add it to my own installation. And I will try to catch the error condition. I can give you feedback tomorrow.

Thanks! Let me know :) would be good to catch some JSON files so we can eventually add tests.

File: pyoverkiz/client.py [hunk 1]
```diff
@@ -809,7 +809,7 @@ async def check_response(response: ClientResponse) -> None:
             ) from error
 
         if result.get("errorCode"):
-            message = result.get("error").strip("'")
+            message = result.get("error").strip('"')
```
Comment (tetienne) [inline pyoverkiz/client.py:812]: If we don’t add any test, it would be nice to add as comment an example of why we trip the ".

Comment (iMicknl) [reply pyoverkiz/client.py:812]: Agree! Preferably both 😄.

**Comment by shypike** [comment]
It looks like this one is the only deviation:

```
{"error":"\"No registered event listener.\"","errorCode":"UNSPECIFIED_ERROR"}
```

**Comment by shypike** [comment]
I've also tested the same issue with the cloud connection.
Conclusion.
Cloud says:
```
{"errorCode":"UNSPECIFIED_ERROR","error":"No registered event listener"}
```
Local says:
```
{"error":"\"No registered event listener.\"","errorCode":"UNSPECIFIED_ERROR"}
```

**Comment by iMicknl** [comment]
Thanks @shypike. Do you know which HTTP code these requests have?

**Comment by shypike** [comment]
I've added the error codes to the original message,
Of course the required change is not just stripping with .strip("\"'.") but also removing the trailing "." from the string comparisons.

**Comment by iMicknl** [comment]
Thanks! @shypike, I implemented it in #870.

**Pull request closed without merging**